src/index.html

```diff
@@ -1,14 +1,65 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+
+      <div class="header-logo">
+        <a href="*" class="logo">
+          <img src="/src/images/logo.png" alt="Moyo logo">
+        </a>
+      </div>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="*" class="nav__link first is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="*" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="*" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="*" class="nav__link"
+            data-qa="hover">Laptops & Computers
+            </a>
```

Comment:
```suggestion
            <a 
              href="*" 
              class="nav__link"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
```
Pay attention to the formatting